articles/architecture-scenarios/b2c/user-provisioning.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Consumer Identity and Access Management User Provisioning
+description: Understand how user provisioning works in your CIAM implementation. 
+toc: true
+topics:
+    - b2c
+    - ciam
+    - user-migration
+    - identity-store
+contentType: concept
+useCase:
+  - user-provisioning
+---
+# Consumer Identity and Access Management User Provisioning
+
+User provisioning defines how users will be added to your system. This part of the implementation determines where you will store your users' profile information, how you will migrate existing users, and how your users will sign up. 
+
```

Comment:
User provisioning is the process of adding users to your system. When you define your user provisioning process, you'll determine:

* How you will store your users' profiles
* How you will migrate existing users to your new system
* How new users will sign up

In this section, we will walk you through the things you should consider when making these decisions.